src/scripts/src/components/container.js

```diff
@@ -59,7 +59,7 @@ export default class Container {
    */
   shouldFlip(dropdownPos, windowHeight = getWindowHeight()) {
     if (dropdownPos === undefined) {
-      return false;
+      return true;
```

Comment:
Does this actually solve the problem? It seems this change would cause the dropdown to always flip irrespective of if needs to? 🤔

Reply:
Yes, it solved the problem and doesn't cause the dropdown to always flip (as far as I tested), I'll attach the demonstration video here:
[choices-ie10.zip](https://github.com/jshjohnson/Choices/files/1689150/choices-ie10.zip)